code/game/objects/items/toys.dm

```diff
@@ -1570,243 +1413,386 @@
 	icon_state = "nuketoy"
 
 /obj/random/figure/item_to_spawn()
-	return pick(subtypesof(/obj/item/toy/figure))
+	return pick(subtypesof(/obj/item/toy/figure/crew))
 
 
 /obj/item/toy/figure
 	name = "Non-Specific Action Figure action figure"
 	desc = "A \"Space Life\" brand... wait, what the hell is this thing?"
 	icon = 'icons/obj/toy.dmi'
 	icon_state = "nuketoy"
+	w_class = WEIGHT_CLASS_TINY
 	var/cooldown = 0
-	var/toysay = "What the fuck did you do?"
+	var/cooldown_time = 3 SECONDS
 
-/obj/item/toy/figure/New()
+/obj/item/toy/figure/attack_self(mob/user)
 	..()
+	if(cooldown < world.time)
+		cooldown = world.time + cooldown_time
+		activate(user)
+	else
+		on_cooldown(user)
+
+/obj/item/toy/figure/proc/activate(mob/user)
+	return
+
+/obj/item/toy/figure/proc/on_cooldown(mob/user)
+	return
+
+/obj/item/toy/figure/crew
+	var/toysay = "What the fuck did you do?"
+
+/obj/item/toy/figure/crew/Initialize(mapload)
+	. = ..()
 	desc = "A \"Space Life\" brand [name]"
```

Comment:
Could you remove this line? All the figurines actually do have descriptions, but they get replaced by this.

Reply:
Oh, interesting didn't even notice that when making the PR. Will change!